inc/README.md

```diff
@@ -280,59 +274,37 @@ $phone_desc = array( 'Description of the phone number', );
-                    'params' => array(
-                        'rows' => 5,
-                        'cols' => 100,
-                    ),
-                    'meta_key' => 'desc',
-                ),
-            ),
             'params' => array(
-                'init_num_forms' => 1,
-                'max_num_forms' => 6,
+            	'repeated' => array(
+                	'min' => 1,
+                    'max' => 3
+                ),
             ),
-            'meta_key' => 'articles',
         ),
     );
 ?>
 ```
-As you can see, we have two fields in the metabox's `fields` array; one `text` field for the main page's header and another `formset` field for each article's header and description. The formset typed field has a `fields` array with more fields in it. Those fields are repeated according to the parameter's set in the `params` array. The `init_num_forms` is what controls how many of the formsets are displayed by default and `max_num_forms` is how many formsets there are total.
-
-Every field type is supported for this use. That means that you could declare a _formset within a formset_. Think of each of those articles from the example before. If we wanted to have at least 2 but at most 3 links associated with each article, we could declare a formset as one of the fields of the article formset.
+The `params` array has an array set called `repeated` with two pairs named `min` and `max` that hold the minimum required and maximum possible fields. *Note*: `min` must be less than or equal to `max`.
 
-The **naming convention** for formset fields are how each of the fields are saved and retrieved uniquely from the database. The `meta_key` is key that the data is associated with. The name of each field of a formset is as such: `<formset meta_key>_<formset iteration integer>_<field meta_key`. So the `meta_key` of each field of the formset in the above example would be `articles_0_headline`, `articles_0_desc`, `articles_1_headline`, `articles_1_desc`, etc until the max number of formsets. This means that _without the `meta_key`, formsets will not work at all_.
+_Every field type is supported for this use, even `fieldset`._
 
-**Note**: It is important to note that in the example the fields of the metabox's `fields` array are associated with a string. (In the formset example, there were two fields; one field was called `main_header` and the other was called `articles`.) This is not supported in the formset fields array. As above, each field in the `fields` array in the formset is an unnamed array.
+The **naming convention** for fields in the front-end is how each of the fields are declared. If a text field is declared in a fieldset, then the `key` of both fields will be concatenated to provide a unique key, for example: 
+`<fieldset-key>_<field-key`. For repeated fields, the `key` of the field will be concatenated with the number of which it corresponds, for example: 
+`<field-key>_<repeated-number>`.
```

Comment:
For completeness' sake, would it make sense to show what the resulting key looks like for a field inside a repeated fieldset? And maybe even for a repeated field inside a repeated fieldset?